### PR TITLE
feat: Optimize configuration items, add etcd.prefix and schema for dashboard

### PR DESCRIPTION
**Pull request by dickens7**
#111

### CODE REVIEW

File: charts/apisix-dashboard/templates/configmap.yaml [hunk 1]
```diff
@@ -60,3 +63,7 @@ data:
           password: {{ .password }}
         {{- end }}
     {{- end }}
+  {{ if .Values.config.schema }}
+  schema.json: |-
+{{ .Values.config.schema | indent 4 }}
+  {{ end }}
```
Comment (tokers) [inline charts/apisix-dashboard/templates/configmap.yaml:69]: Missing the EOS symbol.

File: docs/en/latest/apisix-dashboard.md [hunk 2]
```diff
@@ -31,10 +31,42 @@ $ helm repo update
...
 To uninstall/delete the `apisix-dashboard` release:
 
 ```shell
 $ helm uninstall apisix-dashboard
 ```
+
+First Header | Second Header
+------------ | -------------
+Content from cell 1 | Content from cell 2
+Content in the first column | Content in the second column
```
Comment (starsz) [inline docs/en/latest/apisix-dashboard.md:51]: Hello. What's this?

File: docs/en/latest/apisix-dashboard.md [hunk 3]
```diff
@@ -31,10 +31,42 @@ $ helm repo update
...
+ config.conf.etcd.prefix | apisix config's prefix in etcd | /apisix
+ config.conf.etcd.username | ignore etcd username if not enable etcd auth | nil
+ config.conf.etcd.password | ignore etcd password if not enable etcd auth | nil
+ config.conf.log.errorLog.level | supports levels, lower to higher: debug, info, warn, error, panic, fatal | warn
+ config.conf.log.errorLog.filePath | supports relative path, absolute path, standard output | /dev/stderr
+ config.conf.log.accessLog.filePath | supports relative path, absolute path, standard output | /dev/stdout
+ Config.authentication |  |  
+ config.authentication.secert | secret for jwt token generation. | secert
+ config.authentication.expireTime | jwt token expire time, in second | 3600
+ config.authentication.users | username and password for login `manager api` |
```
Comment (nic-chen) [inline docs/en/latest/apisix-dashboard.md:70]:
```suggestion
 config.authentication.users | username and password for login `Manager API` | 
```

File: docs/en/latest/apisix-dashboard.md [hunk 4]
```diff
@@ -31,10 +31,42 @@ $ helm repo update
...
+ config.conf.etcd.password | ignore etcd password if not enable etcd auth | nil
+ config.conf.log.errorLog.level | supports levels, lower to higher: debug, info, warn, error, panic, fatal | warn
+ config.conf.log.errorLog.filePath | supports relative path, absolute path, standard output | /dev/stderr
+ config.conf.log.accessLog.filePath | supports relative path, absolute path, standard output | /dev/stdout
+ Config.authentication |  |  
+ config.authentication.secert | secret for jwt token generation. | secert
+ config.authentication.expireTime | jwt token expire time, in second | 3600
+ config.authentication.users | username and password for login `manager api` | 
+ Config.schema |  |  
+ config.schema | Configure the custom plug-in JSONSchema |
```
Comment (nic-chen) [inline docs/en/latest/apisix-dashboard.md:72]: It should be the entire JSONSchema, not just the custom plug-in JSONSchema

File: docs/en/latest/apisix-dashboard.md (hunk 2 above)
Comment (dickens7) [reply docs/en/latest/apisix-dashboard.md]: fixed

File: charts/apisix-dashboard/templates/configmap.yaml (hunk 1 above)
Comment (dickens7) [reply charts/apisix-dashboard/templates/configmap.yaml]: fixed

File: docs/en/latest/apisix-dashboard.md (hunk 4 above)
Comment (dickens7) [reply docs/en/latest/apisix-dashboard.md]: This configuration parameter description has been optimized

**Review by nic-chen — APPROVED** [review]

File: docs/en/latest/apisix-dashboard.md [hunk 5]
```diff
@@ -31,10 +31,38 @@ $ helm repo update
...
 
 ```shell
 $ helm uninstall apisix-dashboard
 ```
+
+## Configuration
+
+Parameter | Description | Default
+------------ | ------------- | ------------- 
+ Config.conf |  |
```
Comment (tokers) [inline docs/en/latest/apisix-dashboard.md:52]: `config`?

File: docs/en/latest/apisix-dashboard.md [hunk 6]
```diff
@@ -31,10 +31,38 @@ $ helm repo update
...
+ config.conf.etcd.username | ignore etcd username if not enable etcd auth | nil
+ config.conf.etcd.password | ignore etcd password if not enable etcd auth | nil
+ config.conf.log.errorLog.level | supports levels, lower to higher: debug, info, warn, error, panic, fatal | warn
+ config.conf.log.errorLog.filePath | supports relative path, absolute path, standard output | /dev/stderr
+ config.conf.log.accessLog.filePath | supports relative path, absolute path, standard output | /dev/stdout
+ Config.authentication |  |  
+ config.authentication.secert | secret for jwt token generation. | secert
+ config.authentication.expireTime | jwt token expire time, in second | 3600
+ config.authentication.users | username and password for login `Manager API` | 
+ Config.schema |  |
```
Comment (tokers) [inline docs/en/latest/apisix-dashboard.md:66]: Ditto.

File: docs/en/latest/apisix-dashboard.md [hunk 7]
```diff
@@ -31,10 +31,38 @@ $ helm repo update
...
+ config.conf.listen.host | the address on which the `Manager API` should listen. | 0.0.0.0
+ config.conf.listen.port | The port on which the `Manager API` should listen. | 9000
+ config.conf.etcd.endpoints | upports defining multiple etcd host addresses for an etcd cluster | [apisix-etcd:2379]
+ config.conf.etcd.prefix | apisix config's prefix in etcd | /apisix
+ config.conf.etcd.username | ignore etcd username if not enable etcd auth | nil
+ config.conf.etcd.password | ignore etcd password if not enable etcd auth | nil
+ config.conf.log.errorLog.level | supports levels, lower to higher: debug, info, warn, error, panic, fatal | warn
+ config.conf.log.errorLog.filePath | supports relative path, absolute path, standard output | /dev/stderr
+ config.conf.log.accessLog.filePath | supports relative path, absolute path, standard output | /dev/stdout
+ Config.authentication |  |
```
Comment (tokers) [inline docs/en/latest/apisix-dashboard.md:62]: Ditto.

File: docs/en/latest/apisix-dashboard.md (hunk 5 above)
Comment (dickens7) [reply docs/en/latest/apisix-dashboard.md:52]: Here is a separator to separate configuration items

**Review by tokers — APPROVED** [review]

**Pull request closed without merging**